Manager/BackupManager.php

```diff
@@ -77,28 +77,13 @@ public function execute()
         try {
             // If we catch an exception or not, we would still like to try cleaning up after us
             $this->logger->info('[dizda-backup] Cleaning up after us.');
-            $this->processor->cleanUp();
+            $this->processorManager->cleanUp();
         } catch (IOException $e) {
-            $this->logger->error('[dizda-backup] Cleaning up failed.');
+            $this->logger->error('[dizda-backup] Cleaning up failed.', array('exception' => $e));
 
             return false;
         }
 
         return $successful;
     }
-
-    public function getClientManager()
-    {
-        return $this->cm;
-    }
-
-    public function getDatabaseManager()
-    {
-        return $this->dbm;
-    }
-
-    public function getProcessorManager()
```

Comment:
I understand. I did not see you PR until today that's why I did not make the comment before it was merged. 

I'm pro the use case but I think the method is wrong. We will try to find a better method in 4.0.